<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaPrinc</class>
 <widget class="QMainWindow" name="ventanaPrinc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyM Seguros &amp; Servicios</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 75, 84);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_facturacion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 207, 150);
color: rgb(106, 108, 94);</string>
    </property>
    <property name="text">
     <string>FACTURACIÓN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clientes">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 207, 178);
color: rgb(106, 108, 94);</string>
    </property>
    <property name="text">
     <string>CLIENTES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_servicios">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 186, 175);
color: rgb(106, 108, 94);</string>
    </property>
    <property name="text">
     <string>SERVICIOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reportes">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 206, 122);
color: rgb(106, 108, 94);</string>
    </property>
    <property name="text">
     <string>REPORTES</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>531</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 221, 215);</string>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::LongDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BIENVENIDOS AL SISTEMA DE GESTIÓN DE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(168, 168, 168);</string>
    </property>
    <property name="text">
     <string>Autor: Imenzon Carlos A. / Correo: imenzoncarlos97@gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CALENDARIO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(195, 195, 195);</string>
    </property>
    <property name="text">
     <string>CyMSeguros&amp;Servicios</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_caja">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 186, 175);
background-color: rgb(255, 251, 198);
color: rgb(106, 108, 94);</string>
    </property>
    <property name="text">
     <string>CAJA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuARCHIVO">
    <property name="title">
     <string>ARCHIVO</string>
    </property>
    <addaction name="archivo_conectar"/>
    <addaction name="separator"/>
    <addaction name="archivo_copia_seguridad"/>
   </widget>
   <addaction name="menuARCHIVO"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="archivo_conectar">
   <property name="text">
    <string>¿Como cambiar la BD?</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
  </action>
  <action name="archivo_copia_seguridad">
   <property name="text">
    <string>CREAR COPIA DE SEGURIDAD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
